dask_cuda/tests/test_dask_cuda_worker.py

```diff
@@ -67,3 +67,34 @@ def test_rmm_pool(loop):  # noqa: F811
                 memory_resource_type = client.run(rmm.mr.get_default_resource_type)
                 for v in memory_resource_type.values():
                     assert v is rmm._lib.memory_resource.CNMemMemoryResource
+
+
+def test_nvlink_no_rmm_warning(loop):  # noqa: F811
+    os.environ["CUDA_VISIBLE_DEVICES"] = "0,1"
+    try:
+        with popen(["dask-scheduler", "--port", "9379", "--no-dashboard"]):
+            with popen(
+                [
+                    "dask-cuda-worker",
+                    "127.0.0.1:9379",
+                    "--host",
+                    "127.0.0.1",
+                    "--enable-nvlink",
+                ],
+                stdout=True,
+                stderr=True,
+            ) as proc:
+                with Client("127.0.0.1:9379", loop=loop) as client:
+                    assert wait_workers(client, n_gpus=2)
+
+                # grab first 5 lines of dask-cuda-worker startup
+                lines = []
+                for idx, line in enumerate(proc.stderr):
+                    lines.append(line)
+                    if idx == 5:
+                        break
+
+                assert any(b"When using NVLink we" in line for line in lines)
```

Comment:
Can we try moving this to the first test? Perhaps it's something weird with environment variables being carried over by pytest. Note you'll also need to add lines 84-86 to that first test as well.

Reply:
Good suggestion